<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>精确匹配</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QGroupBox" name="dir_select">
          <property name="title">
           <string>选择参与比较的目录</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="listWidget"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>添加</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="delButton">
               <property name="text">
                <string>删除</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="text">
                <string>开始比较</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pauseButton">
               <property name="text">
                <string>暂停</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopButton">
               <property name="text">
                <string>结束</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="loadButton">
               <property name="text">
                <string>加载</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="filePathLabel">
          <property name="text">
           <string>文件路径:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="filePathLineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="hash_select">
          <property name="title">
           <string>选择哈希算法</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="Md4">
             <property name="text">
              <string>Md4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Md5">
             <property name="text">
              <string>Md5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha1">
             <property name="text">
              <string>Sha1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha224">
             <property name="text">
              <string>Sha224</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha256">
             <property name="text">
              <string>Sha256</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha384">
             <property name="text">
              <string>Sha384</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha512">
             <property name="text">
              <string>Sha512</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha3_224">
             <property name="text">
              <string>Sha3_224</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha3_256">
             <property name="text">
              <string>Sha3_256</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha3_384">
             <property name="text">
              <string>Sha3_384</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Sha3_512">
             <property name="text">
              <string>Sha3_512</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="filePathLabelTip">
          <property name="text">
           <string>文件路径：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="resultWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>384</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
